recipes/pydenticon/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pydenticon" %}
+{% set version = "0.3.1" %}
+{% set python_min = "3.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/azaghal/pydenticon/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 67d0da2ee4204b6a460f1af2a739524e3d64b54e9d23b2ce979b0ec519aa65dd
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install -vv ."
+  run_exports: 
+    - {{ pin_subpackage('pydenticon', max_pin="x.x") }}
+
+requirements:
+  host:
+    - python >={{ python_min }}
+    - setuptools
+    - pip
+  run:
+    - python >={{ python_min }}
+    - pytest
+    - conda-forge::pillow
```

Comment:
You don't need to specify the `conda-forge::` channel. We only use it so you can't get it from anywhere else.